app/models/branding_group.rb

```diff
@@ -12,6 +12,7 @@ class BrandingGroup
   field :feature_flags, type: Hash, default: {}
   field :external_link_url, type: String
   field :external_link_description, type: String
+  field :public, type: Boolean, default: false
```

Comment:
Not a blocker, but I think it would be good to create a migration to backfill the data for `public`.  Mongoid is weird with "default" values and pre-existing documents.  You can load the document and call the field and you will get the default value if it hasn't been set, but you cannot query on the value as it _has not been set in the document_.  It is provided by the ODM after the document is loaded and instantiated as an instance of the model.

I'm not calling it a blocker since what we really care about are branding groups where `public` is `true`, and that will only happen if the owner specifically sets that.  Also, new branding groups will get that field populated when saved for the first time.  But if we ever want to go back and find it where `public` is explicitly `false`, we'll have an issue.

Reply:
ah, that's a great point -- will update